package.json

```
@@ -83,6 +83,7 @@
     "remark-gfm": "^4.0.0",
     "spdx-correct": "^3.1.1",
     "swagger-ui-react": "^5.11.2",
+    "thirdweb": "^1.0.0-alpha-11ea9f3ca-20240220200238",
```

good idea to always keep this pinned for now
```suggestion
    "thirdweb": "1.0.0-alpha-11ea9f3ca-20240220200238",
```